dev/release/rat_exclude_files.txt

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+.gitignore
+LICENSE
+NOTICE
+CONTRIBUTING.md
+CHANGELOG.md
+build/**
+dist/**
+.git/**
+cmake_modules/**
+.cmake/**
+test/resources/**
+*.avro
+*.json
+*.parquet
```

Comment:
We want to avoid binary files in the release distribution.

Reply:
I thought this was not allowed per ASF, but I can't find the source. Since rust also contains binaries, let's continue with this for now.